### PR TITLE
don't quota events.k8s.io events by default

### PR DESCRIPTION
**Pull request by Abirdcfly**
Signed-off-by: Abirdcfly <fp544037857@gmail.com>

#### What type of PR is this?
/kind bug

#### What this PR does / why we need it:
see https://github.com/kubernetes/kubernetes/pull/110888
updated from https://github.com/kubernetes/kubernetes/pull/89179
#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note
None
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

**Comment by Abirdcfly** [comment]
cc @deads2k for approve

**Comment by Abirdcfly** [comment]
/cc @deads2k 

for approve

**Comment by Abirdcfly** [comment]
/cc @lavalamp

Please take a look here, similar to https://github.com/kubernetes/kubernetes/pull/110888

File: pkg/quota/v1/install/registry.go [hunk 1]
```diff
@@ -38,16 +40,17 @@ func NewQuotaConfigurationForControllers(f quota.ListerForResourceFunc) quota.Co
...
-	{Group: "authorization.k8s.io", Resource: "selfsubjectaccessreviews"}:  {},
-	{Group: "authorization.k8s.io", Resource: "localsubjectaccessreviews"}: {},
-	{Group: "authorization.k8s.io", Resource: "selfsubjectrulesreviews"}:   {},
+	{Group: "", Resource: "bindings"}:                                        {},
+	{Group: "", Resource: "componentstatuses"}:                               {},
+	{Group: authentication.GroupName, Resource: "tokenreviews"}:              {},
+	{Group: authentication.GroupName, Resource: "subjectaccessreviews"}:      {},
+	{Group: authentication.GroupName, Resource: "selfsubjectaccessreviews"}:  {},
+	{Group: authentication.GroupName, Resource: "localsubjectaccessreviews"}: {},
+	{Group: authentication.GroupName, Resource: "selfsubjectrulesreviews"}:   {},
```
[the remark this answers is not in the document]
Comment (Abirdcfly) [reply pkg/quota/v1/install/registry.go:49]: > why did you change authz to authn for the last four of these?

https://github.com/kubernetes/kubernetes/blob/182e0989ecd3b4307e29815c8dd9527aea10f15e/pkg/apis/authentication/register.go#L25

Replace strings with existing constants only

[the remark this answers is not in the document]
Comment (lavalamp) [reply pkg/quota/v1/install/registry.go:49]: I think you used the wrong constant, changing "authorization" to "authentication" for the final four of these. Was that deliberate?

[the remark this answers is not in the document]
Comment (Abirdcfly) [reply pkg/quota/v1/install/registry.go]: > I think you used the wrong constant, changing "authorization" to "authentication" for the final four of these. Was that deliberate?

You are right.
Such a low-level mistake, embarrassing. 囧

**Comment by lavalamp** [comment]
/lgtm
/approve

I think this should probably go into the release but since code is frozen, you probably need someone from the release team to agree with me

**Comment by Abirdcfly** [comment]
/cc @kubernetes/milestone-maintainers 

Request `/milestone v1.25` tag from release team.

**Comment by cici37** [comment]
Since it is a bug fix and been approved and LGTMed, tag milestone label to let it in.
/milestone v1.25

**Pull request merged**